### ISSUE TITLE
Matching method change issue！

### ISSUE THREAD

**Issue by cnu-Dawn** [WHU-USI3DV/FreeReg#24]
Hello, may I ask what is the difference between the invariant features extracted by YOHO and other general features? I tried using deep learning matching methods to train matching feature point pairs, but the loss is difficult to converge. Can you give me some suggestions.Thanks!

**Comment by HpWang-whu** [comment]
Hi @LoveMYChen,
The invariant features are actually the 'same' as other features. No special designs are needed. Maybe some 'overfitting check' is needed to check the training process.

Yours,

**Comment by cnu-Dawn** [comment]
Hello, due to too few matching point pairs, the loss always remains at a high value. I only added keypoints and rotation invariant features to the network. Do I also need to add rotation invariant features to the network?
